### ISSUE TITLE
undeclared dependency on pytest, new release requested

### ISSUE THREAD

**Issue by hexylena** [ResearchObject/ro-crate-validator-py#5]
Hey y'all, just installed the package via pip and saw an error due to pytest, I'd written a long issue about it before discovering https://github.com/ResearchObject/ro-crate-validator-py/commit/e95b1ab663aee761811a78e5b47e7a577b19076c so it's already been fixed, yay! Could a new release be made at some point in the future (no rush from my side.)

**Comment by stain** [comment]
Agreed - have contacted @Logan-174 by email to gain access to the pypi entry.
